src/app/components/challenge/challenge.component.ts

```diff
@@ -50,6 +50,11 @@ export class ChallengeComponent implements OnInit {
    */
   isLoggedIn: any = false;
 
+  /**
+   * If the user is a host of the challenge
+   */
+  isHost: any = false;
```

Comment:
We know the data type is boolean, so please use it specifically rather than `any`.